### PR TITLE
rdls_schema.json: add regex pattern to coordinate system and update d…

### LINKED ISSUE #199: [Schema] Update `.coordinate_system` to recommend either EPSG or ESRI codes
One of the draft examples takes the coodinate system from https://ghsl.jrc.ec.europa.eu/download.php?ds=bu which only appears to offer data in a coordinate system that is defined in ESRI but not EPSG (something the source itself appears to have wrong). Should we simply update the description of `Resource.coordinate_system` from
"The reference co-ordinate system of this resource, using the [EPSG codelist](https://epsg.org)."
to
"The reference co-ordinate system of this resource, using the EPSG or ESRI codes."

I'm not particularly fond of this solution as having a single code source is best but when even the EC's Joint Research Centre are getting confused between EPSG and ESRI maybe we need to include the possibility of both?

### PR DESCRIPTION
**Pull request by odscjen**
…escription

**Related issues**

closes #199 

**Description**

**Merge checklist**

- [x] Update the changelog ([style guide](developer_docs.md#changelog-style-guide))
- [x] Run `./manage.py` pre-commit

If you added or removed a field:

- [ ] Update the `collapse` option of the jsonschema directives for dataset, resource, hazard, exposure, vulnerability and loss on `reference/schema.md`

**Having trouble?**

See [how to resolve check failures](https://github.com/GFDRR/rdl-standard/blob/dev/developer_docs.md#resolve-check-failures).

### CODE REVIEW

File: schema/rdls_schema.json [hunk 1]
```diff
@@ -866,9 +866,10 @@
           "exclusiveMinimum": 0
         },
         "coordinate_system": {
-          "title": "Reference coordinate system",
+          "title": "Coordinate reference system",
           "type": "string",
-          "description": "The reference co-ordinate system of this resource, using the [EPSG codelist](https://epsg.org).",
+          "description": "The co-ordinate reference system of this resource, using the EPSG or ESRI codes from https://epsg.io/. Provide both the CRS authority scheme and the code separated by a colon, e.g. EPSG:4326.",
```
Comment (duncandewhurst) [inline schema/rdls_schema.json:871]: https://epsg.io/ isn't an authoritative source, but we can reference it as a useful resource. The authoritative list of EPSG codes is available at https://epsg.org/. The only authoritative list of ESRI codes I can find is at https://github.com/Esri/projection-engine-db-doc, but I don't think that's a particularly helpful page to link to so I just linked the ESRI homepage instead.

```suggestion
          "description": "An identifier string for the coordinate reference system (CRS) used in this resource. The CRS must be identified by a code issued by either [EPSG](https://epsg.org/) or [ESRI](https://www.esri.com/). The identifier string format is {authority}:{code}, e.g. EPSG:4326. [https://epsg.io/](https://epsg.io/) provides a searchable database of EPSG and ESRI coordinate system codes."
```

**Review by duncandewhurst — COMMENTED** [review]
Happy for this to be merged if my suggestion is accepted.

File: schema/rdls_schema.json (hunk 1 above)
Comment (stufraser1) [reply schema/rdls_schema.json:871]: Could use https://spatialreference.org/ref/esri/ as a more useful list of ESRI codes?

Comment (odscjen) [reply schema/rdls_schema.json:871]: @stufraser1 the issue with that website is the same as for epsg.io, it isn't an authoritative source.

**Review by matamadio — APPROVED** [review]

**Pull request merged**